concepticondata/conceptlists/Teit-1908-693.tsv

```diff
@@ -0,0 +1,694 @@
+Teit-1908-693-342	342		female [caribou]			66
+Teit-1908-693-343	343		calf [caribou]			66
+Teit-1908-693-344	344	302	moose			66
+Teit-1908-693-345	345	303	rabbit	1136	RABBIT	66
+Teit-1908-693-346	346		small timber rabbit			66
+Teit-1908-693-347	347		snowshoe rabbit			66
+Teit-1908-693-348	348		jack rabbit			66
+Teit-1908-693-349	349		small run socks & timber S&E Columbia			66
+Teit-1908-693-350	350	304	marten	3739	MARTEN	66
+Teit-1908-693-351	351	305	fisher			66
+Teit-1908-693-352	352	306	mink			66
+Teit-1908-693-353	353	307	wolverine			66
+Teit-1908-693-354	354	308	marmot	3719	MARMOT	66
+Teit-1908-693-355	355		mountain [marmot]			66
+Teit-1908-693-356	356		common [marmot]			66
+Teit-1908-693-357	357		hoary [marmot]			66
+Teit-1908-693-358	358	309	antlers			66
+Teit-1908-693-359	359	310	tail	1220	TAIL	66
+Teit-1908-693-360	360	311	hair of animal	580	FUR	66
+Teit-1908-693-361	361	312	hoof	152	HOOF	66
+Teit-1908-693-362	362	313	claw	72	CLAW	66
+Teit-1908-693-363	363	314	bluebird			66
+Teit-1908-693-364	364	315	bluejay			66
+Teit-1908-693-365	365	316	blackbird			66
+Teit-1908-693-366	366	317	redwinged [blackbird]			66
+Teit-1908-693-367	367	318	catbird			66
+Teit-1908-693-368	368	319	crow	913	CROW	68
+Teit-1908-693-369	369	320	mourning dove			68
+Teit-1908-693-370	370	321	duck all kinds	1360	DUCK	68
+Teit-1908-693-371	371	322	golden eagle			68
+Teit-1908-693-372	372	323	white headed eagle			68
+Teit-1908-693-373	373		baldhead [eagle]			68
+Teit-1908-693-374	374	324	meadow lark			68
+Teit-1908-693-375	375	325	robin	2402	ROBIN	68
+Teit-1908-693-376	376	326	blue grouse			68
+Teit-1908-693-377	377	327	prairie chicken			68
+Teit-1908-693-378	378	328	willow grouse			68
+Teit-1908-693-379	379	329	fool hen			68
+Teit-1908-693-380	380	330	goose	1187	GOOSE	68
+Teit-1908-693-381	381	331	swan	2837	SWAN	68
+Teit-1908-693-382	382	332	crane	1830	CRANE	68
+Teit-1908-693-383	383	333	heron	758	HERON	68
+Teit-1908-693-384	384	334	hummingbird	1589	HUMMINGBIRD	68
+Teit-1908-693-385	385	335	hawk	307	HAWK	68
+Teit-1908-693-386	386	336	small red hawk			68
+Teit-1908-693-387	387	337	night hawk or goatsucker			68
+Teit-1908-693-388	388	338	bat	1793	BAT	68
+Teit-1908-693-389	389	339	king bird			68
+Teit-1908-693-390	390	340	king fisher	2444	KINGFISHER	68
+Teit-1908-693-391	391	341	loon			68
+Teit-1908-693-392	392	342	swallow	897	SWALLOW	68
+Teit-1908-693-393	393		blueback [swallow]			68
+Teit-1908-693-394	394	343	magpie	2039	MAGPIE	70
+Teit-1908-693-395	395	344	owl	735	OWL	70
+Teit-1908-693-396	396	345	raven	2543	RAVEN	70
+Teit-1908-693-397	397	346	snipe			70
+Teit-1908-693-398	398	347	chickadee			70
+Teit-1908-693-399	399	348	ptarmigan			70
+Teit-1908-693-400	400	349	hudson bar bird			70
+Teit-1908-693-401	401	350	ruffed grouse			70
+Teit-1908-693-402	402	351	buzzard	1865	BUZZARD	70
+Teit-1908-693-403	403	352	chicken	1318	CHICKEN	70
+Teit-1908-693-404	404	353	egg	744	EGG	70
+Teit-1908-693-405	405	354	shell of egg	598	SHELL	70
```

Comment:
Unmap since SHELL refers to mollusks.

Reply:
Alright, I was in doubt here because of multiple heterogeneous concepts mapped to it. I'll unmap it here and open an issue then.